build/Settings.cs

```diff
@@ -75,7 +75,7 @@ private static string config(string @default = null, [CallerMemberName] string k
 
         public static readonly string ProjectTemplates = $"https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/{ProjectTemplatesVersion}";
 
-        public static string BuildNumber => config(null, "Build.BuildId") ?? config("9999", "APPVEYOR_BUILD_NUMBER");
+        public static string BuildNumber => config(null, "devops_buildNumber") ?? config("9999", "APPVEYOR_BUILD_NUMBER");
```

Comment:
Where did Build.BuildId come from? Can we just make devops_buildNumber == Build.BuildId so it stays the same? That way we can kick off from AppVeyor if we need? Or is this something else?

Reply:
The issue with build.BuildId is that we cannot start from a particular number and it also resets it 0 in certain conditions (not fully sure about this) since we already have a count built up in app veyor we need to start from there. That is why i set up devops_buildnumber